apisix/plugins/openid-connect.lua

```diff
@@ -156,6 +156,10 @@ local schema = {
                     description = "Comma separated list of hosts that should not be proxied.",
                 }
             },
+        },
+        authorization_params = {
+            description = "Extra authorization params to the authorize endpoint",
+            type = "object"
```

Comment:
Could you add test cases for this option?

Reply:
You can refer to these test cases: https://github.com/apache/apisix/blob/master/t/plugin/openid-connect.t